# CS300 Fall 2022

## Name: 

Add Your Name Here

## Date: 

26 Sept 2022

## Reflections

Please write below this line to address the above-mentioned points. Replace the word, `TO DO` with your response. This assignment will be about 100 words of writing.

## Part 1 Questions

+ 1. Please include a copy and pasted listing of your FASTA data.

```
TODO
```

+ 2. Why do you think we are studying this data together? What is common to all these GeneBank IDs? (For example, do they have similar descriptions or functions?)

```
TODO
```

+ 3. What is the name of each gene? Please organize your answer to have the GeneBank ID next to the name of the gene. (Hint: Names can be found in the nucleotide database output.)

```
TODO
```

+ 4. When you click on a result in GenBank, you will likely see a page of information about the sequence including descriptions, functions, and details about the research groups that sequenced the gene. In terms of its informational content, how is the FASTA format different? What advantages are there in having FASTA formatted data files for research projects?

```
TODO
```

## Part 2 Questions

+ 5. Include a copy and pasted view of your alignment output from your DNA sequences below.

```
TODO
```

+ 6. Describe these results: What can we learn from them?

```
TODO
```

+ 7. Going by the knowledge that `-` characters indicate that sequences do not match, which DNA sequence (give the name) appears to have the most mismatches and shares the least common material with the others? 

```
TODO
```

+ 8. Which sequences are most similar to each other in your multiple sequence alignment results?

```
TODO
```

+ 9. After click on the *Phylogenetic Tree* information from your results, what kind of results did you see? What can you learn from these results?

```
TODO
```

+ 10. Using the Phylogenetic Tree, which sequences appear to be most similar and least similar according to the tree? (Hint, you can use the branch lengthto infer distances.) Can you explain the results from the tree?

```
TODO
```

+ 11. Describe the output of *MView*. For example, what information does this output give you?

```
TODO
```

+ 12. What can you learn about the alignments of your sequences?

```
TODO
```

+ 13. What does "Cov" or "Pid" indicate? Note: you are invited to use online search engines to research this detail.

```
TODO
```

## Part 3 Questions

+ 14. After you compared the original DNA sequence with the one you prepared from the reverse translation tool, what were your observations about similarity between the sequences? Were you surprised to make the discovery that you did?

```
TODO
```

+ 15. After studying the Phylogenetic tree to note the levels of relatedness between the sequences, what did you note about the relationship to the others by the reverse-translated sequence? Please explain a potential reason for this discovery.

```
TODO
```

16. How has this activity helped you to prepare for using analytical online tools? 

```
TODO
```


(Did you remember to add you name at the top of this document?)

